README.md

```diff
@@ -9,3 +9,19 @@ Why we manage them in a separate repository are:
 
 - Parameter maintainers don't require write-access to `launcher` repository.
 - To rollback only parameters independently from `launcher`.
+
+## Directory Structure
+<pre><code>
```

Comment:
It would be better to use the [Markdown code block syntax](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks) <code>```</code> instead of the HTML tags.

Reply:
OK. I use ```